README.md

```diff
@@ -10,18 +10,20 @@ We welcome and appreciate all contributions. Got questions or want to discuss so
 
 # Venafi Collection for Ansible
 
-This collection, called `machine_identity`, uses Venafi Trust Protection Platform (TPP) or Venafi as a Service (VaaS)
-to provide certificates and keys to Ansible solutions involving machine identities.
+This collection, called `machine_identity`, uses [Venafi Trust Protection Platform](https://www.venafi.com/platform/trust-protection-platform) or [Venafi as a Service](https://www.venafi.com/venaficloud) to provide keys and certificates to Ansible solutions that require machine identity management.
 
 If you like this collection, please give us a rating on [Ansible Galaxy](https://galaxy.ansible.com/venafi/machine_identity).
 
+<!-- DW: what's a collection? Is that different than a solution, the term we used with Terraform? -->
+
 ## Requirements
 
 Review the [Venafi](https://github.com/Venafi/vcert-python#prerequisites-for-using-with-trust-protection-platform)
 prerequisites, then install Ansible and [VCert-Python](https://github.com/Venafi/vcert-python) (v0.11.2 or higher) using `pip`:
 ```sh
 pip install ansible vcert --upgrade
 ```
+<!-- DW: so requirements are the same for both tpp and vaas? If so, would be helpful to say that here to avoid confusion -->
```

Comment:
The installation requirements are the same but the configuration requirements vary by Venafi product.  They're probably best documented [here](https://github.com/Venafi/vcert/blob/master/README-CLI-PLATFORM.md#prerequisites) and [here](https://github.com/Venafi/vcert/blob/master/README-CLI-CLOUD.md#prerequisites).

Reply:
got it; makes sense